packages/dd-trace/test/appsec/index.express.plugin.spec.js

```diff
@@ -9,41 +10,45 @@ const { json } = require('../../src/appsec/blocked_templates')
 const zlib = require('zlib')
 
 withVersions('express', 'express', version => {
-  describe('Suspicious request blocking - query', () => {
-    let port, server, requestBody
+  describe('Suspicious request blocking - path parameters', () => {
+    let port, server, paramCallbackSpy
 
     before(() => {
       return agent.load(['express', 'http'], { client: false })
     })
 
     before((done) => {
       const express = require('../../../../versions/express').get()
-      const bodyParser = require('../../../../versions/body-parser').get()
 
       const app = express()
-      app.use(bodyParser.json())
 
-      app.get('/', (req, res) => {
-        requestBody()
-        res.end('DONE')
+      app.get('/multiple-path-params/:parameter1/:parameter2', (req, res) => {
+        res.send('DONE')
       })
 
-      app.post('/', (req, res) => {
+      const nestedRouter = express.Router({ mergeParams: true })
+      nestedRouter.get('/:nestedDuplicatedParameter', (req, res) => {
         res.send('DONE')
       })
 
-      app.post('/sendjson', (req, res) => {
-        res.send({ sendResKey: 'sendResValue' })
-      })
+      app.use('/nested/:nestedDuplicatedParameter', nestedRouter)
 
-      app.post('/jsonp', (req, res) => {
-        res.jsonp({ jsonpResKey: 'jsonpResValue' })
+      app.get('/callback-path-param/:callbackedParameter', (req, res) => {
+        res.send('DONE')
       })
 
-      app.post('/json', (req, res) => {
-        res.jsonp({ jsonResKey: 'jsonResValue' })
+      const paramCallback = (req, res, next) => {
+        next()
+      }
+
+      paramCallbackSpy = sinon.spy(paramCallback)
+
+      app.param(() => {
+        return paramCallbackSpy
       })
 
+      app.param('callbackedParameter')
+
       server = app.listen(port, () => {
         port = server.address().port
         done()
```

Comment:
do a little templating here, and remove all the ugly localhost and ports from the rest of the test file ?
```suggestion
        port = server.address().port
        axios = Axios.create({ baseUrl: `http://localhost:${port}` })
```